.travis.yml

```diff
@@ -49,13 +49,15 @@ jobs:
     - &qemuboottest
       stage: tests
       script:
-        - sed -i $(($(grep "config HAVE_AGENT" -n src/platform/Kconfig | cut -f1 -d:)+2))"s/default y/default n/" src/platform/Kconfig
+        - sed -i $(($(grep "config HAVE_AGENT" -n src/platform/Kconfig |
+             cut -f1 -d:)+2))"s/default y/default n/" src/platform/Kconfig
         - ./scripts/docker-run.sh ./scripts/xtensa-build-all.sh -r $PLATFORM
         - ./scripts/docker-qemu.sh ../sof.git/scripts/qemu-check.sh $PLATFORM
       env: PLATFORM='byt cht'
       before_install:
         - *docker-pull-sof
-        - docker pull thesofproject/sofqemu && docker tag thesofproject/sofqemu sofqemu
+        - docker pull thesofproject/sofqemu &&
+            docker tag thesofproject/sofqemu sofqemu
```

Comment:
Are we folding based on the 80-character lines or the 100-character lines?

Reply:
The (other...) commit message says: "fix the last yamllint warnings".

Reply:
Ugh, and that one wasn't updated to 100-character limit, eh.